File: BackEnd/Mapeamento.Domain/Entities/PontoTuristico.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Mapeamento.Domain.Entities;

public class PontoTuristico
{
    public int Id {get; set;}

    public string Nome {get; set;} = string.Empty;

    public string Descricao {get; set;} = string.Empty;

    public string Localizacao {get; set;} = string.Empty;

    public string Cidade {get; set;} = string.Empty;

    public int EstadoId {get; set;}
    public virtual Estado Estado {get; set;} = null!;

    public DateTime DataCriacao {get; set;} = DateTime.Now;

    public bool Status {get; set;} = true;
}

File: frontend/src/interfaces/PontoTuristico.ts
export interface Estado{
    id:     number;
    nome:   string;
    sigla:  string;
}

export interface PontoTuristico {
    id:          number;
    nome:        string;
    descricao:   string;
    localizacao: string;
    cidade:      string;
    estadoId:    number;
    estado?:     Estado;
    dataCriacao: string;
}

export interface PontoTuristicoPaginado {
    itens:          PontoTuristico[];
    contaRegistros: number;
    pagina:         number;
    tamanhoPagina:  number;
}